langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/LangLibRegexpTest.java

```diff
@@ -125,4 +125,22 @@ private Object[][] longRegexpFindIndexes() {
                 {"testLongIndexFindAllGroups", 549755813632L},
         };
     }
+
+    @Test(dataProvider = "negativeRegexpEmptyCharClassProvider")
+    public void negativeTestRegexpEmptyCharClass(String functionName) {
+        Object returns = BRunUtil.invoke(negativeTests, functionName);
+        Assert.assertEquals(returns.toString(),
```

Comment:
As I Know, now we do not do assertions on Java side, unless it is really need.
You can move these assertions into Ballerina code

Reply:
I followed the same approach used previously. Do I need to change this? @pcnfernando